README.md

```diff
@@ -15,13 +15,16 @@ A collection of the best CSS resources.
 - [Responsive Web design Certification](https://www.freecodecamp.org/learn/2022/responsive-web-design/)
 - [Scrimba](https://scrimba.com/allcourses?price=free&topic=css)
 - [CSS Portal](https://www.cssportal.com/)
+- [Awesome-CSS Learning](https://github.com/micromata/awesome-css-learning)
```

Comment:
Let's remove this one. It's a bit of a duplication of the work done in this repository.